extension/json/json_serializer.cpp

```diff
@@ -216,4 +223,13 @@ void JsonSerializer::WriteDataPtr(const_data_ptr_t ptr, idx_t count) {
 	PushValue(val);
 }
 
```

Comment:
Add WriteValue function to convert a timestamp to a formatted string using a specified format and add it as a JSON string value to the document.